docs/100-upgrade/migration-guides/3.3-3.4.mdx

```diff
@@ -268,6 +268,26 @@ remove its usage and instead copy the original file entirely and apply your
 override on it.<br /> For more information about this change, see the
 [Extend a route layout guide](/docs/3.x/guides/extend-route-layout).
 
+## Removal of `remix-pwa` in favor of `vite-pwa`
+
+In this version we have replaced `remix-pwa` with `vite-pwa`. Although codemods
+should handle removing those in the code, you will need to remove the packages
+from your project:
+
+```shell
+pnpm remove remix-pwa @remix-pwa/dev @remix-pwa/worker-runtime @remix-pwa/cache @remix-pwa/strategy @remix-pwa/sw
+```
+
+`<LiveReload>` component from must be removed from your project, as
```

Comment:
**issue:** are you sure it's needed? Isn't it cleaned by the codemod?

**suggestion:** if it is -> remove. If it's not, reword as
```suggestion
`<LiveReload>` component from must be removed from your project's `root.tsx` root route, as
```